### ISSUE TITLE
Tool to suggest perltidy options to achieve desired style

### ISSUE THREAD

**Issue by y** [perltidy/perltidy#91]
I've spent a lot of time in the documentation and experimenting with different options to try to produce my desired style.
Would it be possible to come at the problem from the other side? Given a snippet of input code, suggest the options that come close to the desired output.

**Comment by shancock9** [comment]
Yes, that would be really useful. In fact, this has been a long-standing request (https://rt.cpan.org/Ticket/Display.html?id=115169).  It would be a difficult task with absolutely no guarantee of matching some existing code, but maybe somebody would like to try it.  Rather than try to dial in each parameter, I think a more practical approach for such an app would be to assemble a large collection of perltidyrc files from github (there are many) and elswhere and then run a piece of sample code on them.  Then, based on some measure of goodness of fit (that's the hard part!), report which of those is the best fit.  That could then be used as the starting point for further refinement.

**Comment by fzzx00348967xer073QBYXSH** [comment]
@y why the short name? There are a lot other meanings for y and you are flexing to all other github users by having that name, I just don't get it why would you get such name like it makes no sense for you to have such name, so my question is why the short name @y?
